src/Symfony/Component/Intl/DateFormatter/IntlDateFormatter.php

```diff
@@ -168,7 +168,7 @@ public function __construct(?string $locale, ?int $datetype, ?int $timetype, $ti
      * @throws MethodArgumentValueNotImplementedException When $locale different than "en" or null is passed
      * @throws MethodArgumentValueNotImplementedException When $calendar different than GREGORIAN is passed
      */
-    public static function create($locale, $datetype, $timetype, $timezone = null, $calendar = self::GREGORIAN, $pattern = null)
+    public static function create(string $locale, ?int $datetype, ?int $timetype, $timezone = null, int $calendar = self::GREGORIAN, ?string $pattern = null)
```

Comment:
in the constructor, locale can be null, + the `@throws` above mentions null is "accepted"